src/main/java/com/efficio/fieldbook/web/fieldmap/controller/FieldmapController.java

```diff
@@ -186,6 +191,34 @@ public Map<String, String> getFieldMapInfoData() {
 		return result;
 	}
 
+	/**
+	 * Delete field map.
+	 *
+	 * @param form the form
+	 * @param model the model
+	 * @return the string
+	 */
+	@ResponseBody
+	@RequestMapping(value = "/deleteFieldmap/{ids}", method = RequestMethod.DELETE)
```

Comment:
@cheryleigh we are trying to avois adding more services to our fieldmap controllers. 
This one can be the beginning of moving to BMSAPI. If this is too much or out os scope it is ok. 
I would not recomment to send ids separated by coma in the URL but I understand this is how it is done now. If you plan to move it to BMSAPI, I suggest to use POST and send the trialIds in the request body. 
We can work on this later